tap_netsuite/netsuite/transaction_entities.py

```diff
@@ -136,6 +136,25 @@ def get_all_generator(self, page_size=200, last_modified_date=None):
         return self._paginated_search_to_generator(paginated_search=paginated_search)
 
 
+class Locations(ApiBase):
```

Comment:
@AlvaroRaul7 why aren't we using this?

https://github.com/hotgluexyz/netsuite-sdk-py/blob/e04c1d24d5c0c70afaac181b10098f69c2fcb5a4/netsuitesdk/api/locations.py#L5